facebook.js

```diff
@@ -1,78 +1,148 @@
+var peoplesmiling  = chrome.i18n.getMessage("peoplesmiling");
+var peoplestanding  = chrome.i18n.getMessage("peoplestanding");
+var peopleonstage = chrome.i18n.getMessage("peopleonstage");
+var peopleplayingmusicalinstruments = chrome.i18n.getMessage("peopleplayingmusicalinstruments");
+var phone  = chrome.i18n.getMessage("phone");
+var plant  = chrome.i18n.getMessage("plant");
+var selfie  = chrome.i18n.getMessage("selfie");
+var shoes  = chrome.i18n.getMessage("shoes");
+var sitting  = chrome.i18n.getMessage("sitting");
+var sky  = chrome.i18n.getMessage("sky");
+var skyscraper  = chrome.i18n.getMessage("skyscraper");
+
+var sleeping  = chrome.i18n.getMessage("sleeping");
+var smiling  = chrome.i18n.getMessage("smiling");
+var snow  = chrome.i18n.getMessage("snow");
+var standing  = chrome.i18n.getMessage("standing");
+var stripes  = chrome.i18n.getMessage("stripes");
+var suit  = chrome.i18n.getMessage("suit");
+var sunglasses  = chrome.i18n.getMessage("sunglasses");
+var swimming  = chrome.i18n.getMessage("swimming");
+var table  = chrome.i18n.getMessage("table");
+var text  = chrome.i18n.getMessage("text");
+var tree  = chrome.i18n.getMessage("tree");
+var twilight  = chrome.i18n.getMessage("twilight");
+var water  = chrome.i18n.getMessage("water");
+var wedding  = chrome.i18n.getMessage("wedding");
+var playingamusicalinstrument  = chrome.i18n.getMessage("playingamusicalinstrument");
+var peopleplayingsport  = chrome.i18n.getMessage("peopleplayingsport");
+
 const emoji_map = {
-  "1 person": "👤",
-  "2 people": "👥",
-  "3 people": "👥",
-  "4 people": "👥",
-  "5 people": "👥",
-  "6 people": "👥",
-  "7 people": "👥",
-  "baby": "🍼",
-  "beach": "🏖",
-  "beard": "👴",
-  "bedroom": "🛏",
-  "bicycle": "🚲",
-  "camera": "📷",
-  "car": "🚗",
-  "cat": "😺",
-  "child": "👦",
-  "christmas tree": "🎄",
-  "close-up": "👀",
-  "closeup": "👀",
-  "cloud": "☁️",
-  "concert": "🎤",
-  "crowd": "👥",
-  "dancing": "💃",
-  "dessert": "🍰",
-  "dog": "🐶",
-  "drink": "🍹",
-  "eating": "🍽",
-  "eyeglasses": "👓",
-  "fireworks": "🎆",
-  "flower": "🌻",
-  "food": "🍎",
-  "glasses": "🕶",
-  "golf": "🏌️‍",
-  "grass": "🍃",
-  "hat": "👒",
-  "indoor": "🏠",
-  "living room": "🏠",
-  "meme": "👍",
-  "mountain": "🌋",
-  "nature": "🏞",
-  "night": "🌃",
-  "ocean": "🌊",
-  "office": "💼",
-  "one or more people": "👥",
-  "outdoor": "🚵",
-  "people eating": "🍽",
-  "people on stage": "🎤",
-  "people playing musical instruments": "🎸",
-  "people playing sport": "🏀",
-  "people sitting": "⑁",
-  "people smiling": "😂",
-  "people standing": "🕴",
-  "phone": "📱",
-  "plant": "🌿",
-  "playing a musical instrument": "🎸",
-  "selfie": "🤳",
-  "shoes": "👡",
-  "sitting": "⑁",
-  "sky": "☀️",
-  "skyscraper": "🏙",
-  "sleeping": "😴",
-  "smiling": "😋",
-  "snow": "❄️",
-  "standing": "🕴",
-  "stripes": "📶",
-  "suit": "🕴",
-  "sunglasses": "🕶",
-  "swimming": "🏊",
-  "table": "🍽",
-  "text": "🔠",
-  "tree": "🌴",
-  "twilight": "🌃",
-  "water": "💧",
-  "wedding": "💒"
+  [per]: "👤",
```

Comment:
It seems like if we are using a string lookup function (like chrome.18n or something else) all this code could go. It's just unneeded indirection.